lib/internal/Magento/Framework/App/Cron.php

```diff
@@ -77,6 +87,19 @@ public function launch()
     }
 
     /**
+     * @deprecated
+     *
+     * @return \Magento\Framework\App\AreaList
+     */
+    private function getAreaList()
+    {
+        if ($this->areaList === null) {
+            $this->areaList = $this->objectManager->get(\Magento\Framework\App\AreaList::class);
```

Comment:
Please use injection via optional parameter in constructor instead of deprecated method, for example:
```php
public function __construct(
    \Magento\Framework\App\AreaList $arealist = null
) {
    $this->arealist = $arealist ?: $this->objectManager->get(\Magento\Framework\App\AreaList::class);
}
```